CONTRIBUTING.md

```diff
@@ -1,96 +1,232 @@
+
+
+## Stage 1: *Proposal*
+
+An RFC at the *proposal* stage is a solution to a problem with enough fidelity
+to be discussed in detail. It must be backed by a willing *champion*. A
+*proposal*'s goal is to make a compelling case for acceptance by describing
+both the problem and the solution via examples and spec edits. A *proposal*
+should be a pull request.
+
+*Entrance criteria:*
+
+* Identified *champion*
+* Clear explanation of problem and solution
+* Illustrative examples
+* Incomplete spec edits
+* Identification of potential concerns, challenges, and drawbacks
+
+A *proposal* is subject to the same discussion as a *strawman*: ensuring that it
+is well aligned with the *guiding principles*, is a problem worth solving, and
+is the preferred solution to that problem. A *champion* is not expected to have
+confidence in every detail at this stage and should instead focus on identifying
+and resolving issues and edge-cases. To better understand the technical
+ramifications of the *proposal*, a *champion* is encouraged to implement it in a
+AsyncAPI library.
+
+Most *proposals* are expected to evolve or change and may be rejected. Therefore,
+it is unwise to rely on a *proposal* in a production environment. AsyncAPI
+libraries *may* implement *proposals*, though are encouraged to not enable the
+*proposed* feature without explicit opt-in.
+
+
+## Stage 2: *Draft*
+
+An RFC at the *draft* stage is a fully formed solution. There is
+consensus the problem identified should be solved, and this particular solution
+is preferred. A *draft's* goal is to precisely and completely describe the
+solution and resolve any concerns through library implementations. A *draft*
+must be a pull request.
+
+*Entrance criteria:*
+
+* Consensus the solution is preferred
+* Resolution of identified concerns and challenges
+* Precisely described with spec edits
+* Compliant implementation in [AsyncAPI JS Parser](https://www.github.com/asyncapi/parser-js) (might not be merged)
+
+A *proposal* becomes a *draft* when the set of problems or drawbacks have been
+fully considered and accepted or resolved, and the solution is deemed
+desirable. A *draft*'s goal is to complete final spec edits that are ready to
+be merged and implement the *draft* in AsyncAPI libraries along with tests to
+gain confidence that the spec text is sufficient.
+
+*Drafts* may continue to evolve and change, occasionally dramatically, and are
+not guaranteed to be accepted. Therefore, it is unwise to rely on a *draft* in a
+production environment. AsyncAPI libraries *should* implement *drafts* to
+provide valuable feedback, though are encouraged not to enable the *draft*
+feature without explicit opt-in when possible.
+
+
+## Stage 3: *Accepted*
+
+An RFC at the *accepted* stage is a completed solution. According to a spec
+Committer it is ready to be merged as-is into the spec document. The RFC is
+ready to be deployed in AsyncAPI libraries. An *accepted* RFC must be
+implemented in AsyncAPI.js.
```

Comment:
Do you mean by `AsyncAPI.js` AsyncAPI JS Parser?

Reply:
Hahahaha! Good catch.